cli/tsdown.config.ts

```diff
@@ -0,0 +1,12 @@
+import type { UserConfig, UserConfigFn } from 'tsdown/config'
```

Comment:
Let it be bundled by the main tsdown config, so we can have it in the core dist.